cla_public/templates/accessibility-statement.html

```diff
@@ -64,19 +66,24 @@ <h2 class="govuk-heading-m">{{ _('Non-accessible content') }}</h2>
     {% trans %}The content listed below is non-accessible for the following reasons.{% endtrans %}
   </p>
 
-  <h3 class="govuk-heading-m">{{ _('Non compliance with the accessibility regulations') }}</h3>
+  <h3 class="govuk-heading-m">{{ _('Non-compliance with the accessibility regulations') }}</h3>
 
-  <ul class="govuk-list govuk-list--bullet">
-    <li>
-      {% trans %}Users are not able to turn off, adjust or extend the time in which inactivity will result in a loss of data. This fails WCAG 2.1 2.2.1 A (Timing Adjustable).{% endtrans %}
-    </li>
-    <li>
-      {% trans %}Some page elements are not available in Welsh. This fails WCAG 2.1 3.1.1 A (Language of Page).{% endtrans %}
-    </li>
-    <li>
-      {% trans %}When a user enters figures into form fields it is not clear that the field does not accept currency symbols. This fails WCAG 2.1 2.4.6 AA (Headings and Labels).{% endtrans %}
-    </li>
-  </ul>
+  <h4 class="govuk-heading-m">{{ _('Time-out') }}</h4>
+  <p class="govuk-body">{% trans %}Users are not able to turn off, adjust or extend the time in which inactivity will result in a loss of data. This fails WCAG 2.1 2.2.1 A (Timing Adjustable).{% endtrans %}
+  </p>
```

Comment:
Is this right?  They can extend the time - by clicking the green button on the timeout screen.  Notwithstanding the but that you've mentioned elsewhere.